racedisparityaudit/ethnicity-facts-and-figures-publisher
# [Do not merge before testing locally] - Rebase data migrations

@andrew-thox and @thomasridd  can you both try this locally first to verify all is well. 

In theory once checked out you should be able to run migrate upgrade (which will do nothing) to verify that nothing changes. That's what we want and they way to get the migrations to be a no-op for the one migration that is now in the codebase is by using the latest identifier from the old migrations.

The point of getting rid of the old migrations is that we can't run them from scratch in a new environment anyway and therefore there is no need to hold onto them.

For unit tests to run locally you'll need to drop and recreate the test DB as a one off.

To test get dump of latest rd-cms data:

pg_dump --no-acl --no-owner -h ec2-34-253-135-152.eu-west-1.compute.amazonaws.com -U u3887ig7ds8rt6 -t db_page -t db_dimension -t db_upload -t alembic_version --data-only d71bou7s6ffku4 > latest.dump

From your local db delete, alembic_version, db_uploads, db_dimension, db_page

Then load the dumped data:

psql -d rdcms < latest.dump

Then run:

./manage.py db upgrade (nothing should change)

run tests and then grunt and click your way around the site and see that the world has not changed.
